lib/rules/selector-not-notation/README.md

```diff
@@ -0,0 +1,57 @@
+# selector-not-notation
+
+## Options
+
+`string`: `"simple"|"complex"`
+
+### `"simple"`
+
+The following patterns are considered problems:
+
+<!-- prettier-ignore -->
+```css
+:not(input, select) { color: pink; }
```

Comment:
```suggestion
:not(a, div) {}
```

Let's align with [our conventions](https://stylelint.io/developer-guide/rules/#write-tests). There are lots of rules in Stylelint, so we try to keep things consistent to make things easier our users and for when we're maintaining.

I'll update the docs to reflect that we usually use `div` as the secondary type selector.